### ISSUE TITLE
In Admin, the link home redirect to root site.

### ISSUE THREAD

**Issue by multiarts** [mjanssen/Laravel-5-breadcrumbs-bundle#15]
In the case also use the admin, but the Home link for example the route always goes to example.com and not to example.com/admin.
How to solve this?

**Comment by mjanssen** [comment]
Home (the first breadcrumb) refers to the base-url (example.com). The second breadcrumb would be "Admin" in this example. Which would refer to example.com/admin.

Could you please explain your question a bit more and/or provide some code? Are you using `Breadcrumbs::automatic();`?

**Comment by multiarts** [comment]
Yes, I'm using Breadcrumbs::automatic() for both front and admin.

**Comment by mjanssen** [comment]
What would you like to do with the breadcrumbs?

**Comment by multiarts** [comment]
That the admin home redirect to the admin.
From: Home / Category / Music and not Home / Admin / Category / Music

**Comment by mjanssen** [comment]
The `Breadcrumbs::automatic();` builds the breadcrumbs by the current URL. So if your url is `example.com/admin/category/music`, your breadcrumbs will be `Home (example.com) / admin (example.com/admin) / category (example.com/admin/category) / music (example.com/admin/category/music)`.

If you want to build the breadcrumbs in a different order (or leaving out / adding one), I suggest building the breadcrumbs within your constructor or function which is called.

**Comment by mjanssen** [comment]
I could update the package so you'll be able to set a base-url, if you want to.

**Comment by multiarts** [comment]
ok, would be great and thank you for your time and attention, I will study more code and contribute.

Att, John Mello

**Comment by mjanssen** [comment]
You can update your local package to the latest one (composer update). This version contains an option to set the set the home url base (`Breadcrumbs::setHomeUrlBase("admin");`). Remember to set the base before generating the breadcrumbs.
